### PR TITLE
feat: 1XP banner

### PR DESCRIPTION
**Pull request by eddybrando**
**🚨 Important: Do not merge yet! Only merge when scheduled and after replacing the links.**

---

## Changes

This PR adds the 1XP banners to the course pages.

Closes https://github.com/Qiskit/platypus/issues/2158

image

### CODE REVIEW

File: frontend/vue/components/OneXP/OneXPBanner.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    flex-direction: column;
+    gap: $spacing-03;
+
+    @include mq($from: large) {
+      flex-direction: row;
+    }
+  }
+
+  &__copy {
+    font-weight: bold;
```
Comment (techtolentino) [inline frontend/vue/components/OneXP/OneXPBanner.vue:62]: same comment as in another PR, but this seems to be spec'd out to be 600...

File: server/templates/textbook.pug [hunk 2]
```diff
@@ -14,6 +14,8 @@ html(lang=course.locale dir=dir)
       q-the-menu
 
     main.body
+      q-one-xp-banner
+
```
Comment (vabarbosa) [inline server/templates/textbook.pug:18]: for courses with multiple versions, we should get clarification if this is how it should look like (multiple spaced out banners)

image

**Review by vabarbosa — CHANGES_REQUESTED** [review]
thank you @eddybrando looks good. can you get confirmation from design on how to treat the below scenario?

**Review by vabarbosa — DISMISSED** [review]
@eddybrando thank you!!! 
only thing is to remember to update the [URL](https://github.com/Qiskit/platypus/blob/04e5a30b6ece2975863c7548b751da2fcf7e3c5e/frontend/vue/components/OneXP/OneXPBanner.vue#L12) before merging :-)
thanks again

File: frontend/vue/components/OneXP/OneXPBanner.vue (hunk 1 above)
Comment (eddybrando) [reply frontend/vue/components/OneXP/OneXPBanner.vue:62]: Similarly,

> As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.

**Review by techtolentino — APPROVED** [review]
LGTM 🚀 

Thanks @eddybrando

**Review by y4izus — APPROVED** [review]

**Pull request merged**